worker-languagedetection-container/testcases/automated/CAF_4001/inuktitut.yaml.testcase

```diff
@@ -27,43 +27,43 @@ expectedOutputData:
       DetectedLanguage1_Code:
         action: "replace"
         values:
-        - data: "iu"
+        - data: "unknown"
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "99"
+        - data: "78"
```

Comment:
Significant change here, cannot identify inuktitut anymore. If it is not supported then remove this testcase and make sure that inuktitut is not referred to as a supported language.